spring-cloud-dataflow-yarn-build-tests/pom.xml

```diff
@@ -32,19 +32,19 @@
 		<dependency>
 			<groupId>com.sun.jersey</groupId>
 			<artifactId>jersey-core</artifactId>
-			<version>1.9</version>
+			<version>1.19</version>
```

Comment:
1.9 is really needed by hadoop as 1.19 wont work. I'll revert these during a merge.